cluster-up/scale-up.sh

```diff
@@ -17,18 +17,26 @@ CONTAINER_MODE=${CONTAINER_MODE:-"1"}
 
 CONTAINER=$(docker ps | grep kubevirt | grep $KUBEVIRT_PROVIDER | awk '{print $1}')
 
+function download_image {
```

Comment:
since there is that md5 use , i cant use it on the two places
over engineering i think if we try too hard to fix that ?